incubator/hnc/hack/crd_patches/ensure-all-served-for-envtest.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+# Generate CRD manifests in config/crd/envtest. Replace "served: false" with
+# "served: true" in the CRD manifests for envtest integration tests. See bug -
+# https://github.com/kubernetes-sigs/controller-runtime/issues/1154
```

Comment:
We should file our own issue for this to track when it's fixed upstream, and reference it here

Reply:
Done.